applications/nrf5340_audio/doc/user_interface.rst

```diff
@@ -84,77 +89,90 @@ LEDs
++--------------------------+-----------------------------------------------------------------------------------------------------------+---------------------------------------------+------------------+------------+
+| **CHG**                  | Off - Charge completed or no battery connected.                                                           | All                                         |         Y        |      N     |
+|                          +-----------------------------------------------------------------------------------------------------------+                                             +------------------+------------+
+|                          | Solid yellow - Charging in progress.                                                                      |                                             |                  |            |
++--------------------------+-----------------------------------------------------------------------------------------------------------+---------------------------------------------+------------------+------------+
+| **OB/EXT**               | Off - No 3.3 V power available.                                                                           | All                                         |         Y        |      Y     |
+|                          +-----------------------------------------------------------------------------------------------------------+                                             +------------------+------------+
+|                          | Solid green - On-board hardware codec selected.                                                           |                                             |                  |            |
+|                          +-----------------------------------------------------------------------------------------------------------+                                             +------------------+------------+
+|                          | Solid yellow - External hardware codec selected.                                                          |                                             |                  |            |
+|                          | This LED turns solid yellow also when the devices are reset, for the time then pins are floating.         |                                             |                  |            |
++--------------------------+-----------------------------------------------------------------------------------------------------------+---------------------------------------------+------------------+------------+
+| **FTDI SPI**             | Off - No data is written to the hardware codec using SPI.                                                 | All                                         |         Y        |      N     |
```

Comment:
General: To ease readability: Do we need subrows for "y"/"n" within the same LED?

Reply:
No, this can also be solved with tabs, as here: https://github.com/nrfconnect/sdk-nrf/blame/main/samples/bluetooth/peripheral_uart/README.rst#L112